sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient.cs

```diff
@@ -76,10 +77,12 @@ public KeyClient(Uri vaultUri, TokenCredential credential, KeyClientOptions opti
         /// <param name="keyType">The type of key to create. See <see cref="KeyType"/> for valid values.</param>
         /// <param name="keyOptions">Specific attributes with information about the key.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <exception cref="ArgumentException"><paramref name="name"/> is an empty string, or <paramref name="keyType"/> contains no value.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="name"/> is null.</exception>
         public virtual Response<Key> CreateKey(string name, KeyType keyType, KeyCreateOptions keyOptions = default, CancellationToken cancellationToken = default)
```

Comment:
If your adding exceptions to the xml comments for the clients you should probably add RequestFailedException as well as this is possible for any pipeline invocations